scripts/unsupported/revert_sam/SamToUnmappedBams.wdl

```diff
@@ -0,0 +1,150 @@
+task CramToBam {
+  File? input_cram
+  String output_bam_name
+  File? ref_fasta
+  File? ref_fasta_index
+  Int? disk_size
+
+  command {
+    samtools view -1 -h -T ${ref_fasta} -o ${output_bam_name} ${input_cram}
+  }
+
+  runtime {
+    docker: "broadinstitute/genomes-in-the-cloud:2.2.3-1469027018"
+    disks: "local-disk " + disk_size + " HDD"
+    memory: "1 GB"
+  }
+  output {
+    File output_bam = "${output_bam_name}"
+  }
+}
+
+task RevertSam {
+  File input_bam
+  File output_map
+  Int disk_size
+
+  command {
+    java -Xmx1000m -jar /usr/gitc/picard.jar \
+    RevertSam \
+    INPUT=${input_bam} \
+    OUTPUT_MAP=${output_map} \
+    OUTPUT_BY_READGROUP=true \
+    VALIDATION_STRINGENCY=LENIENT \
+    ATTRIBUTE_TO_CLEAR=FT \
+    ATTRIBUTE_TO_CLEAR=CO \
+    SORT_ORDER=coordinate
```

Comment:
can we not have this be `queryname` so we dont have to sort after this?

Reply:
I thought the same thing! As it turns out, RevertSam takes significantly longer to sort all of the data (not in parallel) compared to just outputting the same order that came in and then sorting each read group individually (in parallel).